docs/serving/using-custom-metrics-API.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Using the custom-metrics API"
```

Comment:
Hmm, it doesn't seem like this guide covers how to use the custom-metrics API -- if documentation for accessing the metrics already exists, we should point to it. If it doesn't, maybe we could add it to this guide?

Reply:
True. We don't really need a usage guide at this point though. What we need is documentation about the caveats that the implementation of the custom-metrics API in Knative Serving can bring.

Enabling Serving's custom-metric API implementation is needed to enable a set of features in Knative Serving. Today, this feature is "concurrency based autoscaling by the HPA". In the future this will also be "access to existing metrics from custom autoscaler implementations".

Reply:
Ahh, I see. Thanks!